src/sidebar/util/user-agent.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * Helper methods to identify browser versions and os types
+ */
+
+export const isIE11 = (_window = window) => {
+  return _window.navigator.userAgent.indexOf('rv:11') >= 0;
+};
+
+export const isMacOS = (_window = window) => {
+  return _window.navigator.userAgent.indexOf('Mac OS') >= 0;
```

Comment:
These functions could accept the user agent string as an argument rather than `window`, with a default value of `navigator.userAgent`, since `navigator.userAgent` is a global that exists in all browsers we support (and many much older ones we do not).